src/app/features/asset-list/asset-list.tsx

```diff
@@ -108,6 +109,9 @@ export function AssetsList() {
         )}
       </BitcoinNativeSegwitAccountLoader>
 
+      <CurrentStacksAccountLoader fallback={<AddStacksLedgerKeysItem />}>
```

Comment:
@alter-eggo - can you please check this as it's causing a double display of the `Connect Stacks` button on Ledger if you only have a BTC account:
![DoubleStacks_Ledger](https://github.com/leather-wallet/extension/assets/2938440/110bcc5e-9444-4c1d-a461-0521d6e62fa1)